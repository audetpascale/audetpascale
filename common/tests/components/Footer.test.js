/**
 * @jest-environment jsdom
 */
import React from "react";
import renderer from "react-test-renderer";
import { Footer } from "../../index";

test("Generate Footer", () => {
  const component = renderer.create(
    <Footer author="John Doe" title="Jest" image="Logo here" year="1900" />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
