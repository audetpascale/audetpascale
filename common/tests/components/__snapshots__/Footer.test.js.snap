// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Generate Footer 1`] = `
<footer
  className="css-c015yt-Footer"
>
  <div
    className="css-1yrmf69"
  >
    <span
      className="css-vurnku"
    >
      © 
      1900
       - 
      2021
       
      John Doe
      .
      <span
        className="css-j7wx2a-Footer"
      >
        Logo here
        Jest
         est propulsé par
         
        <a
          href="https://www.gatsbyjs.com/"
        >
          Gatsby
        </a>
         et
         
        <a
          href="https://www.netlify.com/"
        >
          Netlify
        </a>
        .
      </span>
    </span>
  </div>
</footer>
`;
